src/Bridge/Symfony/Bundle/DependencyInjection/Compiler/IdentifierNormalizerPass.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Bridge\Symfony\Bundle\DependencyInjection\Compiler;
+
+use Ramsey\Uuid\Uuid;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\Compiler\PriorityTaggedServiceTrait;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * Injects identifier normalizers.
+ *
+ * @author Antoine Bluchet <soyuka@gmail.com>
+ */
+final class IdentifierNormalizerPass implements CompilerPassInterface
```

Comment:
Can you also make this class `@internal`? When we'll bump the DI component to 4.1, we'll be able to use http://symfony.com/doc/3.4/service_container/tags.html#reference-tagged-services.
Alternatively, we can bump the version now and remove this class.

Reply:
@dunglas do you mean `"^3.4 || ^4.0"`?

Reply:
indeed